idea_town/frontend/static-src/app/views/experiment-page.js

```diff
@@ -185,32 +185,32 @@ export default PageView.extend({
     evt.target.style.width = width + 'px';
     evt.target.classList.add('state-change');
 
-    this.updateAddon(true, this.model);
-
-    app.on('webChannel:addon-install:install-ended', () => {
-      this.model.enabled = !this.model.enabled;
+    // TODO: Should this maybe be an event listener at a global level?
```

Comment:
That sounds like a pretty hefty PR on its own, and would rather get this fix in ASAP